### PR TITLE
Feature/bem updates

### PR DESCRIPTION
**Pull request by nirazul**
A few small updates as a preparation for the css class rewrite:

- Bumps for dependencies
- Improve BEM interfaces

### CODE REVIEW

File: src/components/accordion/accordion.vue [hunk 1]
```diff
@@ -4,8 +4,8 @@
                 class="u-reset ui-accordion--head"
                 role="heading"
                 tabindex="0"
-                v-if="!!$scopedSlots.head"
                 :aria-expanded="state.isOpen ? 'true' : 'false'"
+                v-if="!!$scopedSlots.head"
```
Comment (faebeee) [inline src/components/accordion/accordion.vue]: Isn't this against the stylelint rules within our projects?

File: src/mixins/bem/index.js [hunk 2]
```diff
@@ -1,14 +1,11 @@
...
-            return [];
-        },
-    },
-};
+import { getComputedConfig, getPropsConfig, mapFacets, createBemClass, DEFAULT_OPTIONS } from './helpers';
 
+/**
+ * Add BEM related helpers to a component
+ * @param {string} bemRoot - The block name
+ * @param {IBemMixinOptions} config - The mixin config
```
Comment (faebeee) [inline src/mixins/bem/index.js]: AFAIK config is an optional/nullable parameter

File: src/components/accordion/accordion.vue (hunk 1 above)
Comment (nirazul) [reply src/components/accordion/accordion.vue]: You might be right, but the vue eslint plugin isn't smart enough to catch it. I've fixed it anyway!

File: src/mixins/bem/index.js (hunk 2 above)
Comment (nirazul) [reply src/mixins/bem/index.js]: Fixed

**Pull request merged**